js/components/portal-dashboard/feedback/feedback-legend.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+import { FeedbackLevel } from "../../../util/misc";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-legend.less";
+
+interface IProps {
+  feedbackLevel: FeedbackLevel;
+}
+
+export const FeedbackLegend: React.FC<IProps> = (props) => {
+  const {feedbackLevel} = props;
```

Comment:
spacing (I know it's trivial, but we tend to use the extra spaces when dereferencing props):
`const { feedbackLevel } = props;`